src/package.json

```diff
@@ -24,6 +24,7 @@
     "semver": "^5.5.0",
     "simple-spellchecker": "^0.9.8",
     "underscore": "^1.9.1",
+    "winreg": "^1.2.4",
```

Comment:
can we file a ticket to have all of this dependencies to be exact?

Reply:
@enahum Why do you need them to be exact? The API shouldn't change for 1.x.x, so `^` appears to be correct IMHO.
https://stackoverflow.com/a/22345808/3514658

Reply:
Or maybe you want to have `~` instead?

Reply:
cause we use exact versions for dependencies everywhere and this one should not be the exception ;)

Reply:
@enahum Ok. Let's open a ticket for this asking to bumb dependencies as well (which will require a new testing session, so we are a bit short for the open release of today). Because right, now according to `npm` there are still some deps we use for which we haven't applied updates. In the meantime, I merge this PR :)
```

removed 1 package and audited 25559 packages in 55s
found 34 vulnerabilities (15 moderate, 19 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

Reply:
Yeap, as I said this has to be another ticket which I already created https://mattermost.atlassian.net/browse/MM-15208